src/JabbR-Core/Hubs/Chat.cs

```diff
@@ -737,12 +781,17 @@ void INotificationService.NudgeRoom(ChatRoom room, ChatUser user)
 
         void INotificationService.LeaveRoom(ChatUser user, ChatRoom room)
         {
+
             LeaveRoom(user, room);
         }
 
         private void LeaveRoom(ChatUser user, ChatRoom room)
         {
             var userViewModel = new UserViewModel(user);
+            
+            //TODO Remove explicit hub call
```

Comment:
That is one of the issues we have #122. We have to explicitly make the calls in each of the commands, when normally they would just be triggered by click.